Python/tier2.c

```diff
@@ -2037,6 +2157,7 @@ _PyTier2_Code_DetectAndEmitBB(
         case BINARY_OP:
             CHECK_BACKWARDS_JUMP_TARGET();
             if (oparg == NB_ADD || oparg == NB_SUBTRACT || oparg == NB_MULTIPLY) {
+                SET_BEFORE_BRANCH();
```

Comment:
Note that it doesn't matter here if we set this. Because if the binary op indeed does not need a branch, this will be overwritten with another `SET_BEFORE_BRANCH` elsewhere.